src/lib/server/orders.ts

```diff
@@ -563,7 +563,8 @@ export async function createOrder(
 		!canBeNotified &&
 		paymentMethod !== 'point-of-sale' &&
 		paymentMethod !== null &&
-		params.user.userRoleId !== POS_ROLE_ID
+		params.user.userRoleId !== POS_ROLE_ID &&
+		!params.user.userHasPOSOptions
```

Comment:
Can you please move the whole "if" condition into a variable?

It's really hard to understand what the intent of this check is. This can easily be fixed by moving the condition into a variable and naming the variable accordingly.